src/core/a-entity.js

```diff
@@ -201,6 +203,25 @@ var proto = Object.create(ANode.prototype, {
     }
   },
 
+   /**
+   * @returns {array} Return entity animations.
+   */
+  getAnimations: {
```

Comment:
This might add coupling between animations and entities. Can we have it just so the animation can handle start/stop purely based on `el.stopped`?